docs/sdk-and-tools/erdjs/erdjs-signing-providers.md

```diff
@@ -0,0 +1,409 @@
+
+:::note
+The examples in this section rely on [`@walletconnect/qrcode-modal`](https://www.npmjs.com/package/@walletconnect/qrcode-modal) to build the QR popup.
+:::
+
+In order to create an instance of the provider, do as follows:
+
+```
+import QRCodeModal from "@walletconnect/qrcode-modal";
+import { WalletConnectProvider } from "@elrondnetwork/erdjs-wallet-connect-provider";
+
+var provider;
+
+const bridgeUrl = "https://bridge.walletconnect.org";
+
+const callbacks = {
+    onClientLogin: async function () {
+        QRCodeModal.close();
+        const address = await provider.getAddress();
+        console.log("Address:", address);
+    },
+    onClientLogout: async function () {
+        console.log("onClientLogout()");
+    }
+};
+
+const provider = new WalletConnectProvider(bridgeUrl, callbacks);
+```
+
+Before performing any operation, make sure to initialize the provider:
+
+```
+await provider.init();
+```
+
+### Login and logout
+
+Then, ask the user to login using Maiar on her phone:
+
+```
+const connectorUri = await provider.login();
+QRCodeModal.open(connectorUri);
+```
+
+Once the user confirms the login, the `onClientLogin()` callback (declared above) is executed.
+
+
+In order to logout, do as follows:
+
+```
+await provider.logout();
+```
+
+### Signing transactions
+
+Transactions can be signed as follows:
+
+```
+import { Transaction } from "@elrondnetwork/erdjs";
+
+const firstTransaction = new Transaction({ ... });
+const secondTransaction = new Transaction({ ... });
+
+await provider.signTransactions([firstTransaction, secondTransaction]);
+
+// "firstTransaction" and "secondTransaction" can now be broadcasted.
+```
+
+Alternatively, one can sign a single transaction using the method `signTransaction()`.
+
+### Signing messages
+
+:::important
+Documentation in this section is preliminary and subject to change.
+:::
+
+As of July 2022, erdjs' Wallet Connect provider does not allow one to sign arbitrary messages (only transaction signing is supported).
+
+## The Hardware Wallet (Ledger) Provider
+
+:::note
+Make sure you have a look over [this page](/wallet/ledger), in advance.
+:::
+
+[`@elrondnetwork/erdjs-hw-provider`](https://github.com/ElrondNetwork/elrond-sdk-erdjs-hw-provider) allows the users of a dApp to login and sign transactions using a [Ledger device](/wallet/ledger).
+
+In order to create an instance of the provider, do as follows:
+
+```
+import { HWProvider } from "@elrondnetwork/erdjs-hw-provider";
+
+const provider = new HWProvider();
+```
+
+Before performing any operation, make sure to initialize the provider (also, the Elrond application has to be open on the device):
+
+```
+await provider.init();
+```
+
+### Login
+
+Before asking the user to login using the Ledger, you may want to display all the available addresses on the device, and let the user choose one of them:
```

Comment:
`log in`*

Reply:
also, all the addresses aren't displayed on the device, but rather on the website/application. The device only displays one of them at a time

Reply:
Rephrased.